### ISSUE TITLE
Queue up mass batch changes

### ISSUE THREAD

**Issue by romulasry** [fedora-infra/anitya#1039]
I would like to mass batch change the following:

Backend: Custom to Sourceforge
url/project from remove all https:// up to the Sourceforge name

Do able?

**Comment by Zlopez** [comment]
Yes, you can use the Anitya API for this. Look at the https://anitya.readthedocs.io/en/stable/api.html#post--api-v2-projects-

**Comment by Zlopez** [comment]
Or maybe this will be more to your liking https://anitya.readthedocs.io/en/stable/api.html#post--api-v2-versions-

**Comment by romulasry** [comment]
Any chance for support on the Web frontend?

**Comment by Zlopez** [comment]
This could be done, but it's not a priority.
